network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -379,6 +380,14 @@ def get_context(self, request, *args, **kwargs):
                 language_code=language_code,
             )
 
+        BuyersGuideProductCategory = apps.get_model(app_label="wagtailpages", model_name="BuyersGuideProductCategory")
+        category_cache_key = f"pni_home_categories_{language_code}"
+        categories = cache.get(category_cache_key)
+        if not categories:
+            categories = BuyersGuideProductCategory.objects.filter(hidden=False)
+            categories = localize_queryset(categories)
```

Comment:
Using `localize_queryset` instead of the previous code simplifies the calculation but introduces 2 extra queries. This will be addressed once we look into optimising the query performance.